karma.conf.js

```diff
@@ -59,64 +59,24 @@ if (argv.coverage) {
 
 module.exports = function(config) {
   config.set({
-    hostname: '0.0.0.0',
```

Comment:
This block was mainly alphabetized so it was easier to see which options were in use. The `target` attribute is new.